test/benchee_test.exs

```diff
@@ -253,6 +253,40 @@ defmodule BencheeTest do
     end
   end
 
+  test ".run accepts atom keys for jobs" do
+    capture_io fn ->
```

Comment:
I think right now the [docs](https://hexdocs.pm/ex_unit/ExUnit.CaptureIO.html) for `capture_io` assume that you're capturing the IO for the purposes of inspecting it for an assertion. I think it would be nice to add an option to `ExUnit.Case` so you could do the following:

```elixir
defmodule YourModuleTest do
  use ExUnit.Case, async: true, disable_io: true

  test "the truth" do
    assert 1 == 1
  end
end
```

Under the hood it would just use the same function, but I think adding an actual, documented API would help folks find this solution if they want it.

I guess, though, that this is sort of what [`ExUnit.CaseTemplate`](https://hexdocs.pm/ex_unit/ExUnit.CaseTemplate.html#summary) is for. It just seems like a nice to have thing, but they'll probably just say that `CaseTemplate` is good enough...